source/Hardpoint.cpp

```diff
@@ -112,6 +112,12 @@ bool Hardpoint::IsTurret() const
 
 
 
+bool Hardpoint::IsGun() const
+{
+	return !isTurret;
+}
+
+
```

Comment:
this is useless, just call !IsTurret(), its the same

Reply:
Sure. Just commenting though that Vitalchip's larger codebase has support for missile pylons, which are something akin to the missile-only hardpoints we've discussed on numerous ocasions. This is actually part of the code that checks to ensure that it isn't a gun *or* a pylon. So I left it in for now, but yeah, this will probably be cleaned up if we aren't likely to want missile specific hardpoints anytime in the future.

Reply:
I don't see the harm in having a method like this honestly, it makes reading the code easier

Reply:
Removed for now.

Reply:
Yeah it makes sense, it was only used in one place, if we did smth like this it should be everywhere
you can resolve this